package.json

```diff
@@ -157,5 +158,8 @@
     "*.{json,css,scss,md,yaml, yml}": [
       "prettier -w"
     ]
+  },
```

Comment:
Volta is what I use instead of nvm to automatically manage node versions in projects. I recommend it!